src/N98/Magento/Command/Database/DumpCommand.php

```diff
@@ -243,6 +244,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        if ($input->getOption('exclude') && $input->getOption('include')) {
```

Comment:
Why can't I specify include and exclude together? Should allow me to first exclude a bunch but then add two or three other tables as well.